packages/superset-ui-connection/types/SupersetClient/index.d.ts

```diff
@@ -0,0 +1,76 @@
+type body = RequestInit['body'];
```

Comment:
unsure if it's better for these `type`s to all be `PascalCase` in addition to the `interface`s

Reply:
👍by "regular module" you mean within `src/` and using `import`/`export`? 

If I left it in the global namespace, and someone imported this library, would the behavior be that it would conflict with any global/local type declarations they had similar to a JS global variable?